numpy/core/tests/test_datetime.py

```diff
@@ -1794,21 +1827,22 @@ def test_datetime_is_busday(self):
 
     def test_datetime_y2038(self):
         # Test parsing on either side of the Y2038 boundary
-        a = np.datetime64('2038-01-19T03:14:07Z')
+        a = np.datetime64('2038-01-19T03:14:07')
         assert_equal(a.view(np.int64), 2**31 - 1)
-        a = np.datetime64('2038-01-19T03:14:08Z')
+        a = np.datetime64('2038-01-19T03:14:08')
         assert_equal(a.view(np.int64), 2**31)
 
         # Test parsing on either side of the Y2038 boundary with
         # a manually specified timezone offset
-        a = np.datetime64('2038-01-19T04:14:07+0100')
-        assert_equal(a.view(np.int64), 2**31 - 1)
-        a = np.datetime64('2038-01-19T04:14:08+0100')
-        assert_equal(a.view(np.int64), 2**31)
+        with assert_deprecated():
+            a = np.datetime64('2038-01-19T04:14:07+0100')
+            assert_equal(a.view(np.int64), 2**31 - 1)
+            a = np.datetime64('2038-01-19T04:14:08+0100')
+            assert_equal(a.view(np.int64), 2**31)
 
         # Test parsing a date after Y2038 in the local timezone
```

Comment:
Comment would be more clear as "in the naive timezone." Could do a global search/audit of comments with the phrase "local timezone" and tweak them to be consistent with this pull request's change.